spec/support/authentication_helper.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module AuthenticationHelper
+  def login_as(user)
+    OmniAuth.config.test_mode = true
+    OmniAuth.config.add_mock(:oauth, auth_hash_for(user))
+    Rails.application.env_config['omniauth.auth'] = OmniAuth.config.mock_auth[:oauth]
+    get '/auth/oauth/callback'
```

Comment:
It is just a little change :)
```suggestion
    get auth_oauth_callback_path
```